README.md

```diff
@@ -371,25 +371,3 @@ newPGPSplitMessage, err := pgpMessage.SeparateKeyAndData()
 // Key Packet is in newPGPSplitMessage.GetBinaryKeyPacket()
 // Data Packet is in newPGPSplitMessage.GetBinaryDataPacket()
 ```
-
-### Checking keys
```

Comment:
Maybe instead of deleting the section, say that keys are checked on parsing and that key.Check does nothing (in case people don't read the changelog or code)

Reply:
Done :)